docs/conf.py

```diff
@@ -44,7 +44,12 @@ def root(module):
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, root(snek5000))
+os.environ["NEK_SOURCE_ROOT"] = os.fspath(
```

Comment:
Why something so complicated? i.e. why not
```python
os.environ["NEK_SOURCE_ROOT"] = snek5000.get_nek_source_root()
print("NEK_SOURCE_ROOT = ", os.environ["NEK_SOURCE_ROOT"])
```

Reply:
Only if it is `pip install -e .[docs]`. I am not sure how readthedocs installs.

Reply:
It's easy to require such installation in the file `.readthedocs.yml`. Anyway that wasn't a great idea. Let's keep it like that.